core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreReaderFragment.kt

```diff
@@ -1173,6 +1174,7 @@ abstract class CoreReaderFragment :
     }
   }
 
+  @Suppress("DEPRECATION")
   override fun onOptionsItemSelected(item: MenuItem): Boolean =
```

Comment:
Instead of suppressing this method from the lint. We should use the `MenuProvider` option.